node/service/Cargo.toml

```diff
@@ -5,58 +5,72 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-parking_lot = "0.9.0"
-serde = { version = "1.0.102", features = ["derive"] }
-lazy_static = "1.4.0"
-log = "0.4.8"
-futures = "0.3.4"
-slog = "2.5.2"
-hex-literal = "0.2.1"
-polkadot-primitives = { path = "../../primitives" }
-polkadot-runtime = { path = "../../runtime/polkadot" }
-polkadot-overseer = { path = "../overseer" }
-polkadot-subsystem = { package = "polkadot-node-subsystem", path = "../subsystem" }
-kusama-runtime = { path = "../../runtime/kusama" }
-westend-runtime = { path = "../../runtime/westend" }
-polkadot-rpc = { path = "../../rpc" }
-polkadot-node-core-proposer = { path = "../core/proposer" }
-sp-io = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+# Substrate Client
+authority-discovery = { package = "sc-authority-discovery", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
This is not introduced in this PR, but not a fan of package renames like these, unless there is a clear guideline how to do these renames that is followed by everyone, but there isn't one AFAIK.

Reply:
Agree, I prefer to avoid them also.

Reply:
And why did you keep them?

Reply:
Kept them because it didn't seem worth chasing them down to replace them wherever in the codebase they happened to be. Reordering the dependencies to make them easier to understand is one thing; renaming them is more work.